### PR TITLE
Split docs

### PR DESCRIPTION
**Pull request by Flix6x**

### CODE REVIEW

File: dev/dev.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    docker run --name test-postgres -p 5432:5432 -e POSTGRES_PASSWORD=tbtest -e POSTGRES_USER=tbtest -e POSTGRES_DB_NAME=tbtest -d postgres
+
+And run tests:
+
+    make test
+
+To update dependencies run:
+
+    pip-compile -o ci/requirements.txt -U
```
Comment (nhoening) [inline dev/dev.md:24]: I believe you mean `dev` instead of `ci` here? (other command below, as well)

File: timely_beliefs/beliefs/classes.py [hunk 2]
```diff
@@ -667,7 +667,7 @@ def set_event_value_from_source(
     @property
     def knowledge_times(self) -> pd.DatetimeIndex:
         return pd.DatetimeIndex(
-            self.event_starts.to_series(name="knowledge_time").apply(
+            self.event_starts.to_series(name="knowledge_time", keep_tz=True).apply(
```
Comment (nhoening) [inline timely_beliefs/beliefs/classes.py:670]: These seem to be out of scope for this PR?

File: timely_beliefs/tests/test_df_resampling.py [hunk 3]
```diff
@@ -267,3 +267,13 @@ def test_downsample_once_upsample_once_around_dst(
     )
     assert df_resampled_2.sensor == df.sensor
     assert df_resampled_2.event_resolution == event_resolution_2
+
+
+def test_groupby_preserves_metadata(df_4323: BeliefsDataFrame):
```
Comment (nhoening) [inline timely_beliefs/tests/test_df_resampling.py:272]: Is this out of scope for this PR? (Is there a better PR this could go into")

**Review by nhoening — CHANGES_REQUESTED** [review]

File: timely_beliefs/tests/test_df_resampling.py (hunk 3 above)
Comment (Flix6x) [reply timely_beliefs/tests/test_df_resampling.py]: Sorry, I worked off of the `specify-support-of-pandas-versions` branch for this PR (because that first created the `dev` folder), and should have set the PR to merge with that branch.

File: timely_beliefs/beliefs/classes.py (hunk 2 above)
Comment (Flix6x) [reply timely_beliefs/beliefs/classes.py]: Wrong base branch for merge.

File: dev/dev.md (hunk 1 above)
Comment (Flix6x) [reply dev/dev.md]: Thanks.

**Review by nhoening — APPROVED** [review]
